README.md

```diff
@@ -199,6 +199,12 @@ Struct.new([field1, field2])
 
 Interaction with the Soroban-RPC server is done through the `Soroban.RPC` module.
 
+To interact with another network than the futurenet, you can change the `:stellar_sdk` config, which will also change to the RPC endpoint that works with that network. e.g.
```

Comment:
Currently, Soroban.ex can run on both Futurenet and Testnet, the former being the default. To use the Testnet network change the `:stellar_sdk` configuration, which will also change to the RPC endpoint that works with that network. e.g.

Reply:
How is it going?